README.md

```diff
@@ -0,0 +1,216 @@
+# xylem
+
+Library to interact with CarbonCulture's datastore
+
+
+## Installation
+
+Installation using pip::
+
+    pip install https://github.com/CarbonCulture/xylem/archive/v0.4.11.zip
```

Comment:
I had a little skim, couldn't find any info, but lots of other repos have it. Not worth a proper look into now I guess, as you say.